src/components/TodoContainer.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import TodosList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
+
+
+class TodoContainer extends React.Component {
+    state = {
+        todos: []
+       };
+
+       handleChange = id => {
+        this.setState(prevState => ({
+          todos: prevState.todos.map(todo => {
+            if (todo.id === id) {
+              return {
+                ...todo,
+                completed: !todo.completed,
+              }
+            }
+            return todo
+          }),
```

Comment:
[OPTIONAL] I would have written the selected block like this to keep code short.
```
todos: prevState.todos.map(todo => todo.id === id ? { ...todo, completed: !todo.completed} : todo
```
Not sure if its less readable tough.

Reply:
Thank you